docs/src/less/base/index.less

```diff
@@ -0,0 +1,471 @@
+@desktop: ~"only screen and (min-width: 601px)";
+@medium: ~"only screen and (min-width: 961px)";
+@giant: ~"only screen and (min-width: 1281px)";
+
+/* PYGMENTS/ROUGE */
+
+@import "pygments.less";
+
+span.highlight {
+    background-color: #eee;
+    font-family: Consolas, "Liberation Mono", Menlo, Courier, monospace;
+
+    /* fix for long attributes values that don't wrap right */
+    .s {
+        word-break: break-all;
+        word-wrap: break-word;
+    }
+}
+
+figure.highlight {
+    background-color: #efefef;
+    box-sizing: border-box;
+    font-size: 12px;
+    margin: 8px 0;
+    width: 100%;
+
+    pre {
+        font-family: monospace;
+        margin: 0;
+        padding: 8px 8px 0;
+    }
+
+    code {
+        white-space: pre-wrap;
+    }
+}
+
+/* MAIN LAYOUT */
+
+body {
+    margin: 0;
+}
+
+header {
+    background-color: @color-link-default;
+}
+
+footer {
+    margin-top: 32px;
+}
+
+main > div > h2 {
+    margin-top: 24px;
+    padding-top: 8px;
+}
+
+.page-grid {
+    padding: 8px;
+}
+
+.skin-graphic {
+    box-sizing: border-box;
+    display: block;
+    margin: 48px auto 0 auto;
+    max-width: 600px;
+    width: 100%;
+}
+
+.modules-expander {
+    background: #eee;
+    padding: 4px;
+}
+
+.modules-expander__tip {
+    color: @color-grey5;
+    margin: 4px 8px;
+    text-align: center;
+}
+
+.modules-expander__list {
+    column-count: 2;
+    font-size: 20px;
+    line-height: 1.75em;
+    list-style-type: none;
+    margin: 0;
+    width: 100%;
+}
+
+.modules-expander[open] {
+    padding-bottom: 32px;
+}
+
+.modules-expander__list a {
+    text-decoration: none;
+}
+
+.modules-expander .details__summary {
+    padding: 8px 0;
+}
+
+.app-bar {
+    align-items: center;
+    box-sizing: border-box;
+    color: white;
+    display: flex;
+    justify-content: space-between;
+    margin: 0 8px;
+    padding: 16px 0;
+}
+
+.app-bar__title {
+    font-size: 26px;
+    font-weight: bold;
+    margin: 0;
+}
+
+.app-bar__links {
+    flex-shrink: 0;
+    list-style-type: none;
+    margin-left: 8px;
+    padding: 0;
+}
+
+.app-bar__links li {
+    float: left;
```

Comment:
🤔 Float? Why not a `display: flex` with `flex-direction: row` on the parent?

Reply:
It achieves the same thing really (removing the annoying space between inline list items) - just that float is more ingrained in me after many years of use! I agree though - let's move forward with these "new" layout techniques instead of the old hacks :-) Will change it.